Testing/UnitTests/xASL_ut_function_xASL_im_ResampleLinearFair.m

```diff
@@ -171,7 +171,7 @@
 %% Test run 5
 
 % Give your individual subtest a name
-UnitTest.tests(5).testname = 'Resample 3D (up, down, etc.)';
+UnitTest.tests(5).testname = 'Resample 3D (up, down, old, etc.)';
```

Comment:
No this is wrong. All unit tests should be using the old/slow version as reference I would say. So you run up, and then again down (not separately but in serial, taking the output from up and use it again for down), both compare them with the old/slow version, and then we're fine.

Reply:
As said above, I don't think we should do that and I don't think we should spend even more time on this. Originally I only wanted to clean up some code, then we suddenly started to implement new features because a lot of newer users stumbled upon this function and were annoyed by how slow it is. Now everything's kind of convoluted.

Reply:
Let's shortly discuss this on Monday. But this function is currently only used for VTK export and for resampling 2 colormaps. We should test if old and new are equal or similar and then throw away the old function. There is absolutely no reason to keep it since it is not used in any critical imaging code. Kill your darlings.

Also, we could rename `ResampleLinearFair` to `ResampleLinear`.

Reply:
> Also, we could rename `ResampleLinearFair` to `ResampleLinear`

Agree 👍